pkg/operator/ceph/cluster/cluster.go

```diff
@@ -525,6 +525,17 @@ func (c *cluster) postMonStartupActions() error {
 		return errors.Wrap(err, "failed to create cluster rbd bootstrap peer token")
 	}
 
+	// Delete bootstrap keys
+	// TODO: add additional note on why these keys are deleted.
+	bootstrapKeys := []string{"client.bootstrap-mds", "client.bootstrap-mgr", "client.bootstrap-osd", "client.bootstrap-rbd", "client.bootstrap-rbd-mirror", "client.bootstrap-rgw"}
```

Comment:
`client.bootstrap-osd` is the only one used by Rook as far as I can tell from the code. Others are safe to delete according to the Ceph team.

I see `client.bootstrap-rgw` in the external script, but only in the `DummyRados` class that is (IIRC) used only for unit tests.